py3status/modules/diskdata.py

```diff
@@ -4,16 +4,16 @@
 
 Configuration parameters:
     cache_timeout: refresh interval for this module. (default 10)
-    disk: show stats for disk or partition, i.e. `sda1`. None for all disks.
-        (default None)
+    disks: mountpoints, disks or partitions list to show stats
+        for, i.e. `["sda1", "/home", "/dev/sdd"]`.
+        None for all disks.
```

Comment:
(Fluffy) Get rid of `None for all disks.` line.

Reply:
Cannot recreate '/dev/sda' problem, probably for my disk layout, what is the problem?
Removed None line.

Reply:
Displayed `sda` (yours) vs `/dev/sda` (master) when using `/dev/sda`. ;-)